.circleci/config.yml

```diff
@@ -63,6 +63,14 @@ jobs:
       - run:
           name: SSH Key setup
           command: echo -e "Host *\n\tStrictHostKeyChecking no\n\n" > ~/.ssh/config
+      - run:
+          name: install dockerize
+          command: wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+          environment:
+            DOCKERIZE_VERSION: v0.3.0
+      - run:
+          name: Wait for DB
+          command: dockerize -wait tcp://localhost:3306 -timeout 1m
```

Comment:
Reason for adding this?

Reply:
Can we wait just before installing WP if MySQL docker is not started? Instead of waiting ahead of time?

Reply:
Before this line: https://github.com/vaishaliagola27/ci-demo/blob/develop/.circleci/install_test_wp.sh#L20

Reply:
Done the change. Please check.